packages/tables/docs/03-columns/08-text-input.md

```diff
@@ -35,3 +35,17 @@ use Filament\Tables\Columns\TextInputColumn;
 
 TextInputColumn::make('background_color')->type('color')
 ```
+
+## Lifecycle hooks
```

Comment:
Please also add the relevant documentation for other editable columns

Reply:
Would also be useful to see how the `$record` can be injected. And the `$state`.